opentasks-provider/src/main/java/org/dmfs/provider/tasks/TaskDatabaseHelper.java

```diff
@@ -531,9 +531,9 @@ public final static String createIndexString(String table, boolean unique, Strin
         buffer.append("CREATE ");
         if (unique)
         {
-            buffer.append(" UNIQUE ");
+            buffer.append("UNIQUE ");
         }
-        buffer.append("INDEX ");
+        buffer.append("INDEX IF NOT EXISTS");
```

Comment:
How can this work? This doesn't leave a space between "EXISTS" and the table name.

Reply:
Sorry, the commited version missed the space, the 'IF NOT EXIST ' append was in a separate line when I tested.
This is why the build failed as well. Fixed now.